plugins/pulp_python/plugins/models.py

```diff
@@ -1,120 +1,85 @@
-from gettext import gettext as _
 import hashlib
-import re
-import tarfile
 
 from mongoengine import StringField
 from pulp.server.db.model import FileContentUnit
+import twine  # noqa
 
 from pulp_python.common import constants
 
 
 DEFAULT_CHECKSUM_TYPE = 'sha512'
-
-
-# These are required to be in the PKG-INFO file.
-REQUIRED_ATTRS = ('name', 'version', 'summary', 'home_page', 'author', 'author_email', 'license',
-                  'description', 'platform')
+REQUIRED_ATTRS = ('name', 'version', 'author', 'author_email', 'description', 'summary')
 
 
 class Package(FileContentUnit):
     """
     This class represents a Python package.
     """
 
-    # unit key
     name = StringField(required=True)
     version = StringField(required=True)
-
-    author = StringField()
+    filename = StringField(required=True)
+    author = StringField(required=True)
     author_email = StringField()
     description = StringField()
-    home_page = StringField()
-    license = StringField()
-    platform = StringField()
     summary = StringField()
 
     _checksum = StringField()
     _checksum_type = StringField(default=DEFAULT_CHECKSUM_TYPE)
```

Comment:
I guess it's outside the scope of this PR, but I really wish this was just "checksum" and in the format "<type>:<hex_value>". It's done this way on all the other units, though, so I suppose we should be consistent.